todo-app/public/style.css

```diff
@@ -0,0 +1,49 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background: aliceblue;
+    text-align: center;
+}
+
+ul {
+    list-style: none;
+}
+
+.todosWrapper {
```

Comment:
You're using both camelcase and dashes for your CSS class names here. Would be better to pick one naming convention and stick to it.

Reply:
Yeah you're right.